.github/workflows/ci.yaml

```
@@ -10,38 +10,24 @@ jobs:
+          run_install: false
 
-      - name: Cache Node.js modules
-        uses: actions/cache@v4
+      - name: Install Node.js
+        uses: actions/setup-node@v4
         with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('./package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
+            node-version: 20
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)